evil-maps.el

```diff
@@ -238,7 +238,7 @@
 (define-key evil-motion-state-map "gj" 'evil-next-visual-line)
 (define-key evil-motion-state-map (vconcat "g" [down]) 'evil-next-visual-line)
 (define-key evil-motion-state-map "gk" 'evil-previous-visual-line)
-(define-key evil-motion-state-map (vconcat "g" [up]) 'evil-previous-visual-line)
+(define-key evil-motion-state-map (vconcat "g" [up]) 'evil-visual-previous-line)
```

Comment:
@axelf4 I can't see that this function exists anywhere..?

Reply:
Good catch! I renamed the functions while testing, but accidentally did not change this one back.